json_database/hpm.py

```
@@ -11,9 +11,18 @@
     """HiveMind Database implementation using JSON files."""
     name: str = "clients"
     subfolder: str = "hivemind-core"
+    password: Optional[str] = None
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)